paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -1351,7 +1351,7 @@ public class CoreOptions implements Serializable {
                     .enumType(TimeFieldType.class)
                     .defaultValue(TimeFieldType.SECONDS_INT)
```

Comment:
Can we just set this to `AUTO`? If the field is timestamp, we don't need to set this option.

Reply:
`AUTO` maybe cannot adapt to all types, just added detailed log.